scripts/repeat-process.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
```

Comment:
Can we write automation in this repo in Ruby? Just makes it confusing if we are using both JS and Ruby across the board. Happy to help with this too

Reply:
> This is a POC only - to help me establish a theory...

Reply:
But yes - if it turns out that this works I plan to reimpliment it if possible 👍